posthog/hogql/parse_string.py

```diff
@@ -34,11 +38,19 @@ def parse_string(text: str) -> str:
     text = text.replace("\\a", "\a")
     text = text.replace("\\v", "\v")
     text = text.replace("\\\\", "\\")
-
     return text
 
 
 def parse_string_literal(ctx: ParserRuleContext) -> str:
     """Converts a STRING_LITERAL received from antlr via ctx.getText() into a Python string"""
     text = ctx.getText()
     return parse_string(text)
+
+
+def parse_string_chunk(ctx: ParserRuleContext, escape_quotes=True) -> str:
+    """Converts a STRING_LITERAL received from antlr via ctx.getText() into a Python string"""
```

Comment:
Would be worth explaining in the docstring why this is distinct from both `parse_string` and `parse_string_literal`, and what `escape_quotes` is for

Reply:
I replaced all the names in both `parse_string.py` and `string.cpp` to be:
- `replace_common_escape_characters`
- `parse_string_text_ctx`
- `parse_string_literal_ctx`
- `parse_string_literal_text`